docs/Hooks.md

```diff
@@ -283,12 +284,32 @@ fastify.addHook('preHandler', async (request, reply) => {
 
 You can hook into the application-lifecycle as well.
 
+- [onReady](#onready)
 - [onClose](#onclose)
 - [onRoute](#onroute)
 - [onRegister](#onregister)
 
-<a name="on-close"></a>
+### onReady
+Triggered before the server starts listening for requests. It cannot change the routes or add new hooks.
+Registered hook functions are executed serially. Only after all `onReady` hook functions have completed will the server start listening for requests.
+Hook functions accept one argument: a callback, `done`, to be invoked after the hook function is complete. Hook functions are invoked with `this` bound to the associated Fastify instance.
+
+```js
+// callback style
+fastify.addHook('onReady', (done) => {
+  // Some code
+  this.validatorCompiler === fastify.validatorCompiler
+  done()
```

Comment:
Does this accept an `Error` to abort the server starting?

Reply:
Yes, it is

Fixed the readme 👌